src/ngMock/angular-mocks.js

```diff
@@ -1341,12 +1341,12 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
       // service the ngMockE2E version can pass through to the real backend
       originalHttpBackend = $delegate.$$originalHttpBackend || $delegate;
 
-  function createResponse(status, data, headers, statusText) {
+  function createResponse(status, data, headers, statusText, xhrStatus) {
     if (angular.isFunction(status)) return status;
 
     return function() {
       return angular.isNumber(status)
-          ? [status, data, headers, statusText]
+          ? [status, data, headers, statusText, xhrStatus]
           : [200, status, data, headers];
```

Comment:
well, I could put a hardcoded 'success' there. But as 200 is already hardcoded, I was getting confused whether or not xhrStatus needs to be here.